### PR TITLE
[Bundle] Covering the SetAttributeTabBlockObserver for Bundles by Unit Test

### PR DESCRIPTION
**Pull request by eduard13**
### Description (*)
This PR adds missing unit tests for the following class:
 - `\Magento\Bundle\Observer\SetAttributeTabBlockObserver`

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
N/A

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Bundle/Test/Unit/Observer/SetAttributeTabBlockObserverTest.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        $this->observerMock = $this->createMock(Observer::class);
+        $this->eventMock = $this->getMockBuilder(Event::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getProduct'])
+            ->getMock();
+        $this->productMock = $this->getMockBuilder(Product::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->observer = new SetAttributeTabBlockObserver(
```
Comment (lbajsarowicz) [inline app/code/Magento/Bundle/Test/Unit/Observer/SetAttributeTabBlockObserverTest.php]: Use ObjectManager helper for Unit Tests.

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Just use ObjectManager helper ( `\Magento\Framework\TestFramework\Unit\Helper\ObjectManager` ) and then everything is fine :-)

**Review by lbajsarowicz — APPROVED** [review]
Awesome. Thank you for your contribution.

File: app/code/Magento/Bundle/Test/Unit/Observer/SetAttributeTabBlockObserverTest.php (hunk 1 above)
Comment (eduard13) [reply app/code/Magento/Bundle/Test/Unit/Observer/SetAttributeTabBlockObserverTest.php]: Done 👌

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6511](https://jira.corp.magento.com/browse/ENGCOM-6511) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**